tests/integration_tests/without_db/test_question_answering_udf.py

```diff
@@ -38,7 +42,11 @@ def get_dataframe(self, num_rows='all', start_col=0):
         return return_df
 
 
-def test_question_answering_udf(upload_model_to_local_bucketfs):
+@pytest.mark.parametrize("device_id", [None, 0])
```

Comment:
this would fallback to CPU without a GPU and we wouldn't recognize it. I suggest, to use explicit GPU tests which use skip, because this you would see in the log.

Reply:
I do skip if the current parameter is not None and  there is a available cuda for that (just below the this code block)

Reply:
ok, nevermind, didn't see it